### DISCUSSION TITLE
エディタのプレビューについて、見出しを表示させて欲しい

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#521] [要望・アイデア・不具合]
ご担当者　様

タイトルの通りですが、エディタのプレビューについて、見出しを表示させて欲しいです。
以下に理由を記載します。

現在のプレビューでは、実際の記事の右側に表示される「見出し」が表示されません。
そのため、書いた記事をチェックする場合、全体を俯瞰した見出しのチェックがやりずらいです。

例えばエディタで「プレビューのみ」を選択した場合に、実際の記事の右側に表示される「見出し」を表示するなど、改善していただけると幸いです。

以上です。よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご不便をおかけして申し訳ありません🙏
直近の開発優先度などから考えると先になってしまうと思いますが、検討します！
